### PR TITLE
[release/3.1] Add Fedora 33 runtime ids

### PR DESCRIPTION
**Pull request by omajid**
Fedora 32 is gearing up for release[1], and in-development version of Fedora has been officially labelled as being Fedora 33:

    $ podman run -it fedora:33 cat /etc/os-release
    NAME=Fedora
    VERSION="33 (Container Image)"
    ID=fedora
    VERSION_ID=33
    VERSION_CODENAME=""
    PLATFORM_ID="platform:f33"
    PRETTY_NAME="Fedora 33 (Container Image)"
    ANSI_COLOR="0;34"
    LOGO=fedora-logo-icon
    CPE_NAME="cpe:/o:fedoraproject:fedora:33"
    HOME_URL="https://fedoraproject.org/"
    DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/rawhide/system-administrators-guide/"
    SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
    BUG_REPORT_URL="https://bugzilla.redhat.com/"
    REDHAT_BUGZILLA_PRODUCT="Fedora"
    REDHAT_BUGZILLA_PRODUCT_VERSION=rawhide
    REDHAT_SUPPORT_PRODUCT="Fedora"
    REDHAT_SUPPORT_PRODUCT_VERSION=rawhide
    PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
    VARIANT="Container Image"
    VARIANT_ID=container

[1] https://fedorapeople.org/groups/schedule/f-32/f-32-key-tasks.html

This is a backport of https://github.com/dotnet/runtime/pull/32752

### CODE REVIEW

File: eng/Packaging.props [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <PackageVersion Condition="'$(PackageVersion)' == ''">4.7.0</PackageVersion>
     <!-- major.minor.release version of the platforms package we're currently building
          Pre-release will be appended during build -->
-    <PlatformPackageVersion>3.1.0</PlatformPackageVersion>
+    <PlatformPackageVersion>3.1.1</PlatformPackageVersion>
```
Comment (bartonjs) [inline eng/Packaging.props:30]: @safern Does this require rebuilding the package index with this PR? Or is that something that gets handled automagically?

Comment (omajid) [reply eng/Packaging.props:30]: @safern Ping?

Comment (danmoseley) [reply eng/Packaging.props:30]: @ericstj...@safern is out

Comment (ericstj) [reply eng/Packaging.props:30]: I believe this is sufficient.  @Anipik when do we typically add the stable version to the index?

Comment (Anipik) [reply eng/Packaging.props:30]: we add it after the package has been pushed to the nuget. i.e on the day of release with the branding change.

**Review by bartonjs — APPROVED** [review]

**Review by safern — APPROVED** [review]

**Comment by omajid** [comment]
Can someone please merge this? Thanks

**Comment by ericstj** [comment]
This is for a servicing branch, has this been taken to tactics for servicing approval?

**Comment by ericstj** [comment]
I've added servicing-consider.  This will need to go through shiproom.  FWIW, 3.1 has been locked for a couple weeks so this wouldn't have made it into 3.1.3.  It'll need to be in 3.1.4.

**Comment by omajid** [comment]
FWIW, ship room has previously said that RID-bumps are considered "tell-mode" not "ask-mode".

**Comment by ericstj** [comment]
Got it, I'd expect that too.

**Pull request merged**